core/utils/image_urls.py

```diff
@@ -1,16 +1,30 @@
 from django.conf import settings
-
 from django.utils.http import urlquote
 
+RESIZE = 0
```

Comment:
I could make them strings, but this was my way of emulating an enum in Python so as to make it clearer in the code that I'm passing RESIZE as the server type.

At the least, I can certainly add a comment.